examples/extensions/axpby/axpby.cpp

```diff
@@ -32,9 +32,9 @@ namespace mlx::core {
  *  Follow numpy style broadcasting between x and y
  *  Inputs are upcasted to floats if needed
  **/
-array axpby(
-    const array& x, // Input array x
-    const array& y, // Input array y
+mx::array axpby(
```

Comment:
I'm not sure why this is inside the `mlx::core` namespace. I think it makes more sense to keep it in a top-level / user name space and use the `namespace mx = ...`

Reply:
Do you mind also updating `axbpy.h`?

Reply:
I have fixed the building and updated the extension to use its own namespace.